hknweb/templates/about/abouthkn.html

```diff
@@ -0,0 +1,90 @@
+
+{% block header %}
+<link rel="stylesheet" href="{% static 'css/style.css' %}">
+{% endblock %}
+
+{% block title %}About HKN{% endblock %}
+
+
+{% block heading %}
+About HKN
+{% endblock %}
+
+{% block content %}
+
+<div class='centered'>
+<body>
+  <div class='row'>
+
+    <div class='column left'>
+        <p class='what-about-subtitle'>Eta Kappa Nu (HKN) is the <a href='https://hkn.ieee.org/' target='_blank'>national Electrical and Computer Engineering honor society</a>. Founded in 1915, the Mu chapter at UC Berkeley is among the most active engineering societies at Berkeley. We strive to serve the Electrical Engineering and Computer Science (EECS) community at Berkeley by providing weekly tutoring hours, exam review sessions, department tours, course and staff evaluations, among many other services. Our two offices are located in 290 Cory and 345 Soda near the <a href='https://berkeley.edu/map' target='_blank'>northeastern corner of the UC Berkeley campus</a>.</p>
+      </div>
+      <div class='column right'>
+        <a href="https://hkn.ieee.org/get-involved/awards/outstanding-chapter-award/" target="_blank" align='center'><p>
+          <img id="outstanding-chapter" src="{% static 'img/outstanding_chapter.png' %}" alt="2019 Outstanding Chapter Award" width=250></p>
+          </a>
+      </div>
+      
+  </div>
+
+  <div class="tabset">
+  <!-- Tab 1 -->
+  <input type="radio" name="tabset" id="tab1" aria-controls="services" checked>
+  <label for="tab1">Student Services</label>
+  <!-- Tab 2 -->
+  <input type="radio" name="tabset" id="tab2" aria-controls="indrel">
+  <label for="tab2">Industrial Relations</label>
+  <!-- Tab 3 -->
+  <input type="radio" name="tabset" id="tab3" aria-controls="community">
+  <label for="tab3">Community</label>
+  
+  <div class="tab-panels">
+    <section id="services" class="tab-panel">
+      <div style='padding: 0% 5%;'>
+        <div class='row'>
+        <div class='column half'>
+          <h2>Student Services</h2>
+          
+          <p><strong>Tutoring:</strong> Students who need advice or tutoring in EE and CS courses are encouraged to drop by for free assistance. Officers hold office hours from 11am to 5pm every weekday.</p>
+
+          <p><strong>Exam Preparation:</strong> HKN provides a database of past exams and review problems to prepare students for midterms and finals. In addition, look out for exam review sessions for lower-division courses.</p>
+          
+          <p><strong>Course Surveys and Peer Advising:</strong> HKN conducts course surveys for all classes offered by the EECS department every semester. The results for all professors and TAs who have opted-in are available online. Students who need more advice in choosing classes are encouraged to consult our officers during office hours.</p>
+      </div>
+      <div class='column half'>
+        <p align='right'><img id="maker-workshop" src="{% static 'img/maker workshop.jpg' %}" alt="HKN Maker Workshop" width=500></p>
```

Comment:
Ideally I think you wouldn't have static widths on images like this, otherwise it might look strange or overflow on phones with narrow widths. I'm curious how this looks on an actual phone though as I haven't tested that.